bugsnag.go

```diff
@@ -12,12 +12,10 @@ import (
 	"reflect"
 	"strconv"
 
-	"github.com/fromatob/bugsnack/error"
-	"github.com/fromatob/bugsnack/hashstruct"
-	"github.com/fromatob/bugsnack/internal/stack"
+	"github.com/pkg/errors"
```

Comment:
They add stacktraces to errors, so here is an example: https://github.com/fromAtoB/bugsnack/pull/3/files#diff-828fe330a0739077fef40a69cc0b721eR112